### PR TITLE
AB#316 Fix Heading Bolding Issue and Phone Number

### PR DESCRIPTION
**Pull request by CurtisUnderwood**
### Description

List of proposed changes:

- Fixes two bugs on the home page
- Tabbed headers now bold on mobile
- Service Canada Phone Number corrected

### Additional Notes

### CODE REVIEW

**Review by gregory-j-baker — DISMISSED** [review]

File: frontend/src/pages/home.tsx [hunk 1]
```diff
@@ -241,7 +241,7 @@ const Home: FC = () => {
         </TabContext>
       </section>
 
-      <h2 className="h2">{t('supporting-seniors.title')}</h2>
+      <h2 className="h2 mt-10 mb-8 text-primary-700">{t('supporting-seniors.title')}</h2>
```
Comment (sebastien-comeau) [inline frontend/src/pages/home.tsx:244]:
```suggestion
      <h2 className="h2 text-primary-700">{t('supporting-seniors.title')}</h2>
```

File: frontend/src/pages/learn/index.tsx [hunk 2]
```diff
@@ -314,10 +314,10 @@ const Learn: FC = () => {
       <section>
         {sections.map((section, index) => (
           <div key={index}>
-            <h2 className="h2 text-primary-700">
+            <h2 className="h2 text-primary-700 mt-12">
               {t(`sections.${index}.title`)}
             </h2>
-            <p>{t(`sections.${index}.body`)}</p>
+            <p className='my-6'>{t(`sections.${index}.body`)}</p>
```
Comment (sebastien-comeau) [inline frontend/src/pages/learn/index.tsx:320]:
```suggestion
            <p className='mb-8'>{t(`sections.${index}.body`)}</p>
```

File: frontend/src/pages/learn/index.tsx [hunk 3]
```diff
@@ -314,10 +314,10 @@ const Learn: FC = () => {
       <section>
         {sections.map((section, index) => (
           <div key={index}>
-            <h2 className="h2 text-primary-700">
+            <h2 className="h2 text-primary-700 mt-12">
```
Comment (sebastien-comeau) [inline frontend/src/pages/learn/index.tsx:317]:
```suggestion
            <h2 className="h2 text-primary-700">
```

File: frontend/src/pages/learn/index.tsx (hunk 2 above)
Comment (gregory-j-baker) [reply frontend/src/pages/learn/index.tsx:320]: **Nitpick:** use double quotes

```suggestion
            <p className="my-8">{t(`sections.${index}.body`)}</p>
```

File: frontend/public/locales/en/home.json [hunk 4]
```diff
@@ -20,7 +20,7 @@
         "toll-free": "Canada and the United States Toll-free:",
         "tty": "Canada and the United States TTY:",
         "outside": "Outside Canada and the United States:",
-        "toll-number": "1-800-800-277-9914",
+        "toll-number": "1-800-277-9914",
         "tty-number": "1-800-255-4786",
         "outside-number": "1-613-957-1954 (Call collect)",
```
[the remark this answers is not in the document]
Comment (CurtisUnderwood) [reply frontend/public/locales/en/home.json:25]: I believe we're already using whitespace-nowrap when displaying the numbers

`<MuiLink
                component={Link}
                className="whitespace-nowrap"
                href={t('contact-us.cards.call-us.toll-number-href')}
              >
                {t('contact-us.cards.call-us.toll-number')}
              </MuiLink>`

File: frontend/src/pages/home.tsx (hunk 1 above)
Comment (CurtisUnderwood) [reply frontend/src/pages/home.tsx:244]: Removed

File: frontend/src/pages/home.tsx [hunk 5]
```diff
@@ -263,7 +263,7 @@ const Home: FC = () => {
         />
       </div>
 
-      <h2 className="h2">{t('contact-us.title')}</h2>
+      <h2 className="h2 mt-10 mb-8 text-primary-700">{t('contact-us.title')}</h2>
```
[the remark this answers is not in the document]
Comment (CurtisUnderwood) [reply frontend/src/pages/home.tsx:266]: Removed

File: frontend/src/pages/learn/index.tsx (hunk 3 above)
Comment (CurtisUnderwood) [reply frontend/src/pages/learn/index.tsx:317]: Fixed.

**Review by sebastien-comeau — APPROVED** [review]

**Pull request merged**